main_server/local_server/server_admin/c_comdef_login.php

```diff
@@ -148,7 +148,7 @@
     }
 
     if (!isset($supress_header) || !$supress_header) {
-        echo '<div class="bmlt_admin_logout_bar"><h4><a href="'.$_SERVER['PHP_SELF'].'?admin_action=logout">'.c_comdef_htmlspecialchars($localized_strings['comdef_server_admin_strings']['logout']).'</a></h4>';
+        echo '<div class="bmlt_admin_logout_bar"><h4><a href="'.$_SERVER['PHP_SELF'].'?admin_action=logout">'.c_comdef_htmlspecialchars($localized_strings['comdef_server_admin_strings']['logout']). ($user_obj->GetLocalName() != '' ? ' ('.$user_obj->GetLocalName().')</a></h4>' : '</a></h4>');
```

Comment:
Can you change this so that `</a></h4>` is appended at the end instead of being repeated?